src/utils/ohri-sidebar.ts

```diff
@@ -1,7 +1,9 @@
 export function scrollIntoView(viewId: string) {
-  document.getElementById(viewId).scrollIntoView({
+  const currentElement = document.getElementById(viewId);
+  currentElement?.scrollIntoView({
     behavior: 'smooth',
-    block: 'start',
-    inline: 'start',
+    block: 'nearest',
+    inline: 'nearest',
```

Comment:
@kajambiya I see you changed from 'start' to 'nearest', how does this affect the existing scroll behaviour within pages?

Reply:
> @kajambiya I see you changed from 'start' to 'nearest', how does this affect the existing scroll behaviour within pages?

Didn't check the other pages but actually I noticed it causes no significant difference. Let me just change it back to start.